tests/Feature/SampleFeatureTest.php

```diff
@@ -31,4 +38,288 @@ public function testWithMock()
         self::assertEquals(1, $user1['id']);
         self::assertEquals(2, $user2['id']);
     }
+
+    /**
+     * Test sorting by single column.
+     *
+     * @param Collection $resultData Collection of data
+     * @param string $sortingField Sorting field
+     * @param \Exception|null $expectedException Expected exception
+     *
+     * @dataProvider sortingData
+     */
+    public function testSingleSorting(Collection $resultData, string $sortingField, $expectedException = null)
```

Comment:
I'd better call it `testSortBySingleFieldCheck()` (and, correspondingly `testSortByMultipleFieldsScheck()`)

Reply:
done.